components/gantt/accessibility/overview.md

```diff
@@ -0,0 +1,113 @@
+tags: telerik,blazor,accessibility,wai-aria,wcag,gantt
+slug: gantt-accessibility-overview
+position: 0
+---
+
+# Accessibility Overview
+
+The UI for Blazor Gantt component is <a href="https://www.w3.org/TR/WCAG22" target="_blank">WCAG 2.2 AA</a> and <a href="https://www.section508.gov" target="_blank">Section 508</a> compliant. The component also follows the <a href="https://www.w3.org/WAI/ARIA/apg/" target="_blank">WAI-ARIA best practices</a> for implementing the keyboard navigation for its component <a href="https://www.w3.org/TR/wai-aria/#roles" target="_blank">role</a>, and is tested against the popular screen readers.
+
+# Blazor Gantt Accessibility Example
+
+WCAG 2.2 introduces the <a href="https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements" target="_blank">"Dragging Movements"</a> criterion as an important part of the Operable principle. Its primary goal is to guarantee that any feature reliant on drag actions offers an alternative method that can be executed with a single click, enhancing user accessibility.
+
+The illustrative example below shows the Gantt tree columns reorder action, achievable through the [Column Menu]({%slug gantt-column-menu%}). Telerik UI for Blazor aims to offer a versatile API that allows users to trigger all functions programmatically or externally, meeting diverse accessibility requirements for any applications.
+
+The following example demonstrates the [accessibility compliance of the Gantt component]({%slug gantt-wai-aria-support%}). The described level of compliance is achievable with the [Ocean Blue A11y Accessibility Swatch]({%slug themes-accessibility-swatch%}).
+
+>caption Gantt accessibility compliance example
+
+````CSHTML
+@*Evaluate the example with Axe Core or other accessibility tools*@
+
+<TelerikGantt Data="@Data"
+              Width="900px"
+              Height="600px"
+              IdField="Id"
+              ParentIdField="ParentId"
+              Navigable="true"
+              ColumnReorderable="true"
+              Sortable="true"
+              ColumnResizable="true"
+              ShowColumnMenu="true">
+    <GanttViews>
+        <GanttDayView></GanttDayView>
+        <GanttWeekView></GanttWeekView>
+        <GanttMonthView></GanttMonthView>
+        <GanttYearView></GanttYearView>
+    </GanttViews>
+    <GanttColumns>
+        <GanttColumn Field="@nameof(FlatModel.Title)" ShowColumnMenu="false">
+        </GanttColumn>
+        <GanttColumn Field="@nameof(FlatModel.PercentComplete)">
+        </GanttColumn>
+        <GanttColumn Field="@nameof(FlatModel.Start)">
+        </GanttColumn>
+        <GanttColumn Field="@nameof(FlatModel.End)">
+        </GanttColumn>
+    </GanttColumns>
+</TelerikGantt>
+
+@code {
+    private DateTime SelectedDate { get; set; } = new DateTime(2019, 11, 11, 6, 0, 0);
+
+    public class FlatModel
```

Comment:
The data item class should be at the bottom, unless it's important for understanding this specific example.